dependencies.yaml

```
@@ -98,26 +98,31 @@ dependencies:
               cuda: "11.2"
             packages:
               - cuda-version=11.2
-              - cudatoolkit
           - matrix:
               cuda: "11.4"
             packages:
               - cuda-version=11.4
-              - cudatoolkit
           - matrix:
               cuda: "11.5"
             packages:
               - cuda-version=11.5
-              - cudatoolkit
           - matrix:
               cuda: "11.8"
             packages:
               - cuda-version=11.8
+          - matrix:
+              cuda: "12.0"
+            packages:
+              - cuda-version=12.0
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.*"
+            packages:
               - cudatoolkit
           - matrix:
               cuda: "12.*"
             packages:
```

Think we would want to just relax the constraint

```suggestion
            packages:
              - cuda-version=12
```

---

The `cuda` matrix entry is set to `major.minor` by our CI images / workflows. We want to make sure that we install the same `major.minor` that is requested.